expect.go

```diff
@@ -159,6 +159,15 @@ func SetEnv(env []string) Option {
 	}
 }
 
+// SetSysProcAttr sets the SysProcAttr syscall values for the spawned process.
+func SetSysProcAttr(args *syscall.SysProcAttr) Option {
```

Comment:
How do you call this Option? 

I don't think changing the SysProcAttr after a command has been started has any effect.
If you do SpawnWithArgs with ChangeOption directly after this might work b/c you manage to sneak the changed struct in before the command actually started. 

Does this work for you if you Spawn -> Wait for a bit -> Change options? 

Does it work changing the options back?

Reply:
had a chat with Josh and this was cleared up. 

The option is used as an argument to Spawn.